src/app/visningskomponenter/ytelser/ytelser.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import Grid from "../../utils/grid";
+import {StringOrNull} from "../felles-typer";
+import Aktivitetsfase from "./aktivitetsfase";
+import Innsatsgruppe from "./innsatsgruppe";
+import Rettighetsperiode from "./rettighetsperiode";
+import Status from "./status";
+import Vedtak from "./vedtak";
+import Vedtaksperiode from "./vedtaksperiode";
+
+interface VedtaksperiodeData {
+    fradato: string;
+    tildato: string;
+}
+
+export interface YtelseData {
+    innsatsgruppe: StringOrNull;
+    vedtak: StringOrNull;
+    status: StringOrNull;
+    aktivitetsfase: StringOrNull;
+    vedtaksperiode: VedtaksperiodeData;
+    rettighetsperiode:StringOrNull;
+}
+
```

Comment:
Slik ser en respons ut for kall mot `/veilarboppfolging/api/person/:fnr/ytelser`

```
{
  "vedtaksliste": [
    {
      "vedtakstype": "Ung ufør / Stans",
      "status": "Iverksatt",
      "aktivitetsfase": "Ikke spesif. aktivitetsfase",
      "rettighetsgruppe": "Ingen livsoppholdsytelse i Arena"
    },
    {
      "vedtakstype": "Ung ufør / Ny rettighet",
      "status": "Avsluttet",
      "aktivitetsfase": "Ikke spesif. aktivitetsfase",
      "rettighetsgruppe": "Ingen livsoppholdsytelse i Arena"
    }
  ],
  "ytelser": [
    {
      "status": "Aktiv",
      "ytelsestype": "Arbeidsavklaringspenger",
      "datoMottatt": {
        "year": 2017,
        "month": 7,
        "day": 1
      },
      "datoFra": {
        "year": 2017,
        "month": 1,
        "day": 1
      },
      "datoTil": null
    }
  ],
  "oppfolgingskontrakter": [
    {
      "innsatsgrupper": [
        "Ikke vurdert"
      ],
      "status": "Aktiv"
    }
  ]
}
```